apps/api/src/applicant/endofjourney.service.ts

```diff
@@ -315,25 +315,20 @@ export class EndOfJourneyService implements OnModuleInit {
    * Event handler for delete re-engaged applicants
    */
   @OnEvent(`${SystemMilestoneEvent.REENGAGED}.*`)
-  async handleReEngagedDeleteEvent(
+  async handleReEngagedEvent(
     payload: IENApplicantStatusAudit,
     event: SystemMilestoneEvent,
+    manager: EntityManager,
   ): Promise<void> {
     this.logger.log(`Handling re-engaged event: ${event}`, 'END-OF-JOURNEY');
 
-    const connection = this.connection;
-    if (!connection) {
+    if (!manager) {
       this.logger.error('Connection failed', 'END-OF-JOURNEY');
       return;
     }
-    const queryRunner = connection.createQueryRunner();
-    await queryRunner.startTransaction();
```

Comment:
Why are we removing the transaction here?

Reply:
For delete, update, create milestone in `ienapplicant.service.ts`, they all have `getManager().transaction(async manager)`, so I simply reuse this manager to avoid the stuck issue. I think 2 transactions causes the issue.